### PR TITLE
Adding PSH profile info

### PR DESCRIPTION
**Pull request by psokolhessner**
Added profile information for Peter Sokol-Hessner. 

# Pull Request for profile addition

## Description

Added profile information (links, text) for Peter Sokol-Hessner for use on the Team page. 

## Type of change

- [X] New feature (non-breaking change which adds functionality)

### Requirements Met:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the project documentation
- [X] My changes generate no new warnings or errors
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/TeamProfileCards/index.js [hunk 1]
```diff
@@ -89,20 +89,21 @@ export function ActiveTeamRow() {
           Ari is the creator and core maintainer of Start Your Lab. He is also an incoming Ph.D. student in Social Psychology and a Centennial Fellow at Princeton University (starting Fall 2021). Ari's research focuses on the application of artificial intelligence to facilitate collective problem-solving and collaborative processes on a global scale. With a breadth of experiences across physics, mathematics, finance, and software, he hopes to bring modern tools to the forefront of the social sciences, with a nod toward increased interdisciplinary work between fields and industries.
         </Translate>
       </TeamProfileCardCol>
-      {/*<TeamProfileCardCol
-        name="Peter Sokol-Hessner"
+      <TeamProfileCardCol
+        name="Peter Sokol-Hessner (he/him/his)"
```
Comment (aridyckovsky) [inline src/components/TeamProfileCards/index.js]: This reminds me that we might even want a separate key-value field _just_ for pronouns, which from a responsive web perspective will make the most sense (allows us to style them separately and on different lines if needed for mobile width screens).

Comment (aridyckovsky) [reply src/components/TeamProfileCards/index.js]: I'll need to implement it separately, so no need to do so here prior to review approval.

Comment (psokolhessner) [reply src/components/TeamProfileCards/index.js]: Yes, agreed, had no idea how to do that so did the lame version.

Comment (aridyckovsky) [reply src/components/TeamProfileCards/index.js]: Made a resolution to this with an additional commit, and also specified a new `avatar__pronouns` class in our CSS customization

**Pull request merged**